apps/core/package.json

```diff
@@ -15,50 +15,50 @@
   "dependencies": {
     "@bigcommerce/catalyst-client": "workspace:^",
     "@bigcommerce/components": "workspace:^",
-    "@icons-pack/react-simple-icons": "^9.2.0",
-    "@vercel/analytics": "^1.1.1",
-    "@vercel/kv": "^1.0.0",
-    "@vercel/speed-insights": "^1.0.1",
-    "clsx": "^2.0.0",
-    "gql.tada": "^1.4.0",
+    "@icons-pack/react-simple-icons": "^9.4.0",
+    "@vercel/analytics": "^1.2.2",
+    "@vercel/kv": "^1.0.1",
+    "@vercel/speed-insights": "^1.0.10",
+    "clsx": "^2.1.0",
+    "gql.tada": "^1.4.3",
     "graphql": "^16.8.1",
     "lodash.debounce": "^4.0.8",
-    "lucide-react": "^0.294.0",
+    "lucide-react": "^0.365.0",
     "next": "^14.1.4",
     "next-auth": "5.0.0-beta.4",
-    "next-intl": "^3.9.4",
+    "next-intl": "^3.11.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-google-recaptcha": "^3.1.0",
-    "react-hook-form": "^7.49.3",
+    "react-hook-form": "^7.51.2",
     "react-hot-toast": "^2.4.1",
     "schema-dts": "^1.1.2",
     "server-only": "^0.0.1",
-    "sharp": "^0.33.0",
-    "tailwind-merge": "^2.1.0",
+    "sharp": "^0.33.3",
+    "tailwind-merge": "^2.2.2",
     "zod": "^3.22.4"
   },
   "devDependencies": {
-    "@0no-co/graphqlsp": "^1.7.0",
+    "@0no-co/graphqlsp": "^1.9.1",
     "@bigcommerce/eslint-config-catalyst": "workspace:^",
-    "@gql.tada/cli-utils": "^0.3.0",
+    "@gql.tada/cli-utils": "^0.3.3",
     "@tailwindcss/container-queries": "^0.1.1",
-    "@testing-library/react": "^14.1.2",
+    "@testing-library/react": "^14.2.2",
     "@types/lodash.debounce": "^4.0.7",
-    "@types/node": "^18.17.12",
+    "@types/node": "^20.12.4",
```

Comment:
Should we also change our `.nvmrc` and build pipeline node version too?

Reply:
No (at least not as part of this PR), my bad, didn't plan on updating the types. Good catch 👀